pom.xml

```diff
@@ -67,7 +67,7 @@
     <java.level>8</java.level>
     <jenkins.version>2.222.1</jenkins.version>
     <metrics.version>4.1.6</metrics.version>
-    <revision>4.0.2.9</revision>
+    <revision>4.0.2.11</revision>
```

Comment:
This change should be reverted for the PR. It was for us to change this in order test/build locally, but if we submit a PR to the upstream repo they typically won't want us changing the revisions.